CMakeLists.txt

```diff
@@ -181,7 +181,7 @@ if(HIOP_DEVELOPER_MODE)
   install(TARGETS hiop_options hiop_warnings EXPORT hiop-targets)
 endif()
 
-if(NOT (HIOP_USE_GPU AND HIOP_USE_HIP))
+if(HIOP_USE_RAJA AND (NOT (HIOP_USE_GPU AND HIOP_USE_HIP)))
```

Comment:
Agreed that this closes the xSDK issue, however, we should update in future to fix this based on our discussion.

Reply:
I agree! I saw you have already created an issue for it (#563)